packages/attestation-service/src/metrics.ts

```diff
@@ -61,6 +61,11 @@ export const Counters = {
     labelNames: ['provider', 'country', 'code'],
     help: 'Counter for error code of each failed delivery attempt by provider and country',
   }),
+  attestationRequestsByNumberType: new Counter({
+    name: 'attestation_requests_number_type',
+    labelNames: ['country', 'type'],
+    help: 'Counter for requests by country and type of phone number',
```

Comment:
what do you mean type of phone number? is one of these the prefix and the other the country code itself?

Reply:
ok I've now added string literals so you get ```attestation_requests_number_type{country="US",type="fixed_Line_or_mobile"} 1```